iogt/templates/core/tags/section_listing_homepage.html

```diff
@@ -1,19 +1,64 @@
-{% load wagtailcore_tags static i18n %}
+{% load wagtailcore_tags wagtailimages_tags comments static core_tags i18n %}
 
 {% if sections %}
-<div id="section_menu" class="section-menu">
-  <div class="section-heading">
-    <a href="#"><img src="{% static 'img/iogt_logo_2.png' %}" class="logo" alt="The Internet of Good Things" width="40%" /></a>
-  </div>
-  <ul class="menu-list">
-    {% for section in sections %}
-    <li class="menu-list__item menu-list__item{{section.get_effective_extra_style_hints}}">
-      <a href="{% pageurl section %}">
-        {{section.title}}
-        <span class="menu-list__desc">{{section.description}}</span>
-      </a>
-    </li>
-    {% endfor %}
-  </ul>
-</div>
+<h3 class="heading heading--homepage">{% trans "Recommended" %}</h3>
+<div class="section-list section-list__homepage">
+  {% for section in sections %}
+    {% load_descendant_articles_for_section section featured_in_homepage=True as featured_articles %}
+    {% if featured_articles %}
+    <div class="section-list__item section-list__item{{section.get_effective_extra_style_hints}}">
+      <ul class="article-list">
+      {% for article in featured_articles %}
+      <li class="article-list__item">
+      {% if forloop.first %}
+        <div class="article_thumbnail--hero">
+          <a href="{% pageurl article %}">
+            {% if article.image %}
+              {% image article.image width-320 %}
+            {% endif %}
+          </a>
+        </div>
+        <div class="article_content--hero">
```

Comment:
I'm wary of wrapping things like this; usually when people want to add padding to a whole bunch of things, and that breaks down after awhile.

It's far better to add padding to things individually.